Tools/CISupport/ews-build/steps.py

```diff
@@ -4441,10 +4442,15 @@ def finished(self, results):
         log_text = self.log_observer.getStdout() + self.log_observer.getStderr()
         match = re.search(r'S3 URL: (?P<url>[^\s]+)', log_text)
         # Sample log: S3 URL: https://s3-us-west-2.amazonaws.com/ews-archives.webkit.org/ios-simulator-12-x86_64-release/123456.zip
-        s3url = ''
+
+        self.build.s3url = ''
+        build_url = f'{self.master.config.buildbotURL}#/builders/{self.build._builderid}/builds/{self.build.number}'
         if match:
-            s3url = match.group('url')
-        self.build.s3url = s3url
+            self.build.s3url = match.group('url')
+            print(f'build: {build_url}, url for GenerateS3URL: {self.build.s3url}')
```

Comment:
These logs go to server logs, wouldn't be visible in buildbot UI.

Reply:
Do we actually think anyone will check logs server side? We don't do that much, usually better to put error messages in the steps logs

Reply:
I might check server logs to debug the issue. We don't want to expose this info publicly.